website/pages/docs/plugins/sources/shopify/_meta.json

```diff
@@ -0,0 +1,5 @@
+{
+  "overview": "Overview",
+  "configuration": "Configuration",
```

Comment:
Minor: I think it would be fine to merge the Configuration page into the Overview, so that it's all in one place. We've been doing that more often in some recent plugins.

Reply:
I don't agree, the configuration page is long (documents 6 config keys + example) and the overview page is not very short due to the shopify-wierdness explanations

Reply:
It's not that long (I read through all of it in < 1 minute 😄 ) and you can't use the plugin without seeing the configuration, so it seems important enough to have an example on the overview page.

Reply:
I'm undecided so I'll leave it as you suggested.